apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -340,7 +340,7 @@ class LessonEditor extends Component {
         </CollapsibleEditorSection>
 
         <CollapsibleEditorSection title="Activities & Levels" fullWidth={true}>
-          <ActivitiesEditor />
+          <ActivitiesEditor hasLessonPlan={true} />
```

Comment:
This seems like a great way to break up the launch of this feature into smaller pieces 👍